projects/documentation/src/components/styles.css

```diff
@@ -950,6 +951,10 @@ sp-link:not(:defined) {
     border-radius: 6px;
 }
 
+.deprecation-container {
+    margin: 3em 0;
```

Comment:
The demo content and page header both use `2em` for their spacing, not sure if that's something to take in mind when making the decision on spacing here.